### PR TITLE
[Communication] - New version of Identity Service

### PR DESCRIPTION
**Pull request by RezaJooyandeh**
As a part of this PR. The Identity client is extracted from the Administration package and moved to a new package as well.
ApiView: https://apiview.dev/Assemblies/Review/290c367d508340a386fc4a42d7474f36

Note: To reduce the number of conflicts, I did not rerecord the chat live tests.

### CODE REVIEW

File: sdk/communication/Azure.Communication.Identity/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Azure Communication Identity client library for .NET
+
+> Server Version:
+> Identity client: 2021-03-07
+
+Azure Communication Identity is managing tokens for Azure Communication Services.
+
+[Source code][source] <!--| [Package (NuGet)][package]--> | [Product documentation][product_docs] | [Samples][source_samples]
```
Comment (RezaJooyandeh) [inline sdk/communication/Azure.Communication.Identity/README.md:8]: I had to comment the package link out for now as the package is not published on NuGet yet.

File: sdk/communication/Azure.Communication.Identity/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+[source]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/communication/Azure.Communication.Identity/src
+[source_samples]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Identity/samples
+[cla]: https://cla.microsoft.com
+[coc]: https://opensource.microsoft.com/codeofconduct/
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
+<!--[package]: https://www.nuget.org/packages/Azure.Communication.Identity-->
+[product_docs]: https://docs.microsoft.com/azure/communication-services/overview
+[nuget]: https://www.nuget.org/
+[user_access_token]: https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-csharp
```
Comment (minnieliu) [inline sdk/communication/Azure.Communication.Identity/README.md:133]: Just a note that we will have to update these microsoft docs as well, they refer to the administration package.

Comment (RezaJooyandeh) [reply sdk/communication/Azure.Communication.Identity/README.md:133]: Yes I will filed something in the backlog to update the docs

File: sdk/communication/Azure.Communication.Identity/samples/Sample1_CommunicationIdentityClientAsync.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+```C# Snippet:CreateCommunicationUserAsync
+Response<CommunicationUserIdentifier> userResponse = await client.CreateUserAsync();
+CommunicationUserIdentifier user = userResponse.Value;
+Console.WriteLine($"User id: {user.Id}");
+```
+
+You will need to store the `identity` that is returned by Azure Communication Services. This is necessary for creating tokens or refreshing them in the future and to map your user identities with Azure Communication Services identities. The `identity` value should be treated as a secret.
+
+## Generate a user token## Generate a user token
```
Comment (minnieliu) [inline sdk/communication/Azure.Communication.Identity/samples/Sample1_CommunicationIdentityClientAsync.md:41]: Double title

Comment (RezaJooyandeh) [reply sdk/communication/Azure.Communication.Identity/samples/Sample1_CommunicationIdentityClientAsync.md]: Fixed

**Review by JianpingChen — APPROVED** [review]

**Pull request merged**